docs/intro/using.rst

```diff
@@ -122,8 +122,8 @@ The following listing shows examples with and without transpose operation.
 Reuse FFT plan
 --------------
 
-Apart from the basic APIs, Kokkos-fft APIs include overloaded APIs which can take a FFT plan as an argument.
-Using these overloaded APIs, we can reuse the FFT plan created before. 
+Apart from the basic APIs, Kokkos-fft offers the capability to create a FFT plan wrapping the FFT plans of backend libraries.
+We can reuse the FFT plan created once to perform FFTs multiple times on different data.
```

Comment:
```suggestion
One can reuse the FFT plan created once to perform FFTs multiple times on different data given that they have the same properties.
```
Just a word to say we cannot pass anything ?

Reply:
Fixed accordingly